tests/contract_testcases/evm/error.sol

```diff
@@ -36,3 +36,23 @@ contract c is e {
 
 	}
 }
+
+// ----
```

Comment:
What do you think of the entry header being this?
`// ---- EXPECTED DIAGNOSITCS ----`

Reply:
I've changed it to something similar without the upper case